docs/ros/ros/config/yaml.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: Robot YAML
+sidebar_label: Robot YAML
+sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
```

Comment:
Are we not implementing any content here in the first release?

Reply:
Luis will be adding this section.